tags/environmentalsustainability.md

```diff
@@ -0,0 +1,77 @@
+# CNCF TAG Environmental Sustainability
+
+## Introduction
+
+The Paris Climate Accords outline the link between economic activity, greenhouse gas (GHG) emissions, and the impacts of climate change. GHG  emissions, primarily made from Carbon Dioxide, are released during the combustion of fossil fuels to produce electricity. To reach net zero goals, we need to identify how to reduce the environmental footprints of increasingly large data centers.
+
+## Mission Statement
+This TAG's goal is to advocate for, develop, support, and help evaluate environmental sustainability initiatives in Cloud Native Technologies.
+
+All activities of this TAG should aim to positively impact the community by changing mindset, increasing resource efficiency, and putting energy efficiency and environmental sustainability on everyone's agenda.
+
+## Responsibilities & Deliverables
+
+### Background
+We recognize that: 
+- balancing resource consumption with the desire for performance is difficult
+- there is inconsistency in how emissions are calculated and what is included by software in scopes of emission. 
+
+### In-Scope
+
+- Identify, define, and develop tooling to assess and improve environmental sustainability approaches, including
+    - quantification of energy consumption of full-stack software including application, cloud native services
+    - recommendations to manage full-stack software such that the overall energy consumption and associated carbon are reduced.
+    - items to evaluate technological and architectural health of projects
+- Community outreach on any items within scope, for instance, social media/conferences/podcasts/meetups et cetera
+- Collaboration with other sustainability efforts
```

Comment:
you mean `with other organizations`?

Reply:
We chose "efforts" here, because we must not artificially limit our self with whom or what we can collaborate.

Reply:
@dims  it could be organizations, or it could be projects/universities/??? in other areas.  we chose to be general intentionally.  Is there something else that would be more clear on this?

Reply:
got it. felt incomplete that's all.

Reply:
Any suggestions on what we could add for clarity?

Reply:
@catblade some examples may help